tests/templates/routerTestData/basicEdgeAppDir.ts

```diff
@@ -0,0 +1,236 @@
+					status: 404,
+					data: '<html>404</html>',
+					headers: {
+						'content-type': 'text/html; charset=utf-8',
+						'x-matched-path': '/404',
+					},
+				},
+			};
+		}),
+		...['/', '/index', '/index.rsc', '/api/hello'].map(path => {
+			const fileName = path.replace(/^\/$/, '/index');
+			const matchedPath = path.replace(/^\/index$/, '/');
+
+			return {
+				name: `\`${path}\` returns its page`,
+				paths: [path],
+				expected: {
+					status: 200,
+					data: JSON.stringify({
+						file: fileName.replace(/\.rsc$/, ''),
+						params: [],
+					}),
+					headers: {
+						'content-type': 'text/plain;charset=UTF-8',
+						'x-matched-path': matchedPath,
+					},
+				},
+			};
+		}),
+		{
+			name: '`/` with RSC header returns the page for `/index` and `vary` header',
+			paths: ['/'],
+			headers: { rsc: 'true' },
+			expected: {
+				status: 200,
+				data: JSON.stringify({ file: '/index', params: [] }),
+				headers: {
+					'content-type': 'text/plain;charset=UTF-8',
+					'x-matched-path': '/index.rsc',
+					vary: 'RSC, Next-Router-State-Tree, Next-Router-Prefetch',
+				},
+			},
+		},
+		{
+			name: '`/500` returns the 500 error page',
+			paths: ['/500'],
+			expected: {
+				status: 500,
+				data: '<html>500</html>',
+				headers: {
+					'content-type': 'text/html; charset=utf-8',
+					'x-matched-path': '/500',
+				},
+			},
+		},
+		{
+			name: 'trailing slash redirects to non-trailing slash',
+			paths: ['/api/hello/'],
+			expected: { status: 307, data: '', headers: { location: '/api/hello' } },
+		},
+		{
+			name: 'valid `_next/static` file returns the cache headers',
+			paths: ['/_next/static/chunks/123.js'],
+			expected: {
+				status: 200,
+				data: 'chunk file',
+				headers: {
+					'content-type': 'text/plain;charset=UTF-8',
+					'x-matched-path': '/_next/static/chunks/123.js',
+					'cache-control': 'public,max-age=31536000,immutable',
+				},
+			},
+		},
+		{
+			name: 'invalid `_next/static` file returns 404 and cache headers',
+			paths: ['/_next/static/chunks/invalid-file'],
+			expected: {
+				status: 404,
+				data: '<html>404</html>',
+				headers: {
+					'cache-control': 'public,max-age=31536000,immutable',
+					'content-type': 'text/html; charset=utf-8',
+					'x-matched-path': '/404',
+				},
+			},
+		},
+		{
+			name: 'valid `_next/data` file matches correctly',
+			paths: ['/_next/data/file.json'],
+			expected: {
+				status: 200,
+				data: 'data file',
+				headers: {
+					'content-type': 'text/plain;charset=UTF-8',
+					'x-matched-path': '/_next/data/file.json',
+				},
+			},
+		},
+		{
+			name: 'invalid `_next/data` file returns 404',
+			paths: ['/_next/data/invalid-file'],
+			expected: {
+				status: 404,
+				data: '<html>404</html>',
+				headers: {
+					'content-type': 'text/html; charset=utf-8',
+					'x-matched-path': '/404',
+				},
+			},
+		},
+		{
+			name: 'issue 70 - rewrite phase should enter after filesystem rewrites $1 to $1 with check: true',
```

Comment:
`rewrites $1 to $1`?

Reply:
I guess `$1 to $1` is a little unclear. I can probably improve the test name if you want, perhaps `rewrites a path to the same path`?

For reference, this is the record it relates to.
```
{ src: '/_next/data/(.*)', dest: '/_next/data/$1', check: true }
```

Reply:
ah I see, yeah `rewrites a path to the same path` sounds better to me 😄 

(could you also please add a comment or something there? 🙏 )